### PR TITLE
Fix animation in android theme in dxList selectAll item (T538229)

### PR DESCRIPTION
**Pull request by sometimesElen**

### CODE REVIEW

File: js/ui/list/ui.list.base.js [hunk 1]
```diff
@@ -41,6 +42,10 @@ var ListBase = CollectionWidget.inherit({
 
     _activeStateUnit: LIST_ITEM_SELECTOR,
 
+    _getFeedbackSelector: function() {
```
Comment (lukyanovas) [inline js/ui/list/ui.list.base.js]: I don't think that extracting _getFeedbackSelector  is a good idea - feedback and active state are the same states. Modify _activeStateUnit instead.

**Review by lukyanovas — CHANGES_REQUESTED** [review]

**Review by DokaRus — COMMENTED** [review]
Simple adding .dx-item.dx-list-item classes to the selectAll item didn't fix the issue? Or it has some disadvantages?

**Comment by sometimesElen** [comment]
Adding .dx-item.dx-list-item classes breaks selection, bc list starts to consider selectAll as a simple item.

File: styles/widgets/android5/list.android5.less [hunk 2]
```diff
@@ -54,15 +53,11 @@
         .background-size-prop(100%);
     }
 
-    &.dx-state-active {
+    .dx-inkripple-wave {
```
Comment (DokaRus) [inline styles/widgets/android5/list.android5.less]: `.dx-inkripple-wave` and `&.dx-state-active` use the same background color. 
You can change it to
```
&.dx-state-active,
.dx-inkripple-wave {
  background-color: @ANDROID5_LIST_ITEM_HOVER_BACKGROUND;
}
```

**Review by DokaRus — APPROVED** [review]

**Review by lukyanovas — APPROVED** [review]

**Pull request merged**